src/App.tsx

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
+import { PostList } from './components/PostList/PostList';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const preparedPosts = posts.map(item => (
```

Comment:
```suggestion
const preparedPosts: Post[] = posts.map(item => (
```
You have the `Post` interface